### PR TITLE
Update Razor to work for 3.1 SDKs

### PR DESCRIPTION
**Pull request by NTaylorMullen**
- These bits include a new Razor language server, O# plugin and VSCode extension that update the Razor experience to work on 3.1 DSKs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
   ],
   "defaults": {
     "omniSharp": "1.34.7",
-    "razor": "1.0.0-alpha3-20190819.3"
+    "razor": "1.0.0-alpha3-3.1.0-preview3.19563.3"
```
Comment (filipw) [inline package.json]: new versioning scheme?

Comment (NTaylorMullen) [reply package.json]: Kind of ya. We moved where we build the Razor VSCode extension pieces into AspNetCore-Tooling and as part of that the versioning changed a lot.

**Comment by NTaylorMullen** [comment]
Feel free to merge on my behalf when the repo is ready for it 😄

**Comment by JoeRobich** [comment]
Thanks!

**Pull request merged**